application/src/main/java/org/togetherjava/tjbot/features/mediaonly/MediaOnlyChannelListener.java

```diff
@@ -48,8 +49,7 @@ public void onMessageReceived(MessageReceivedEvent event) {
         if (messageHasNoMediaAttached(message)) {
             message.delete().flatMap(any -> dmUser(message)).queue(any -> {
             }, failure -> tempNotifyUserInChannel(message)
-                .queue(notificationMessage -> notificationMessage.delete()
-                    .queueAfter(1, TimeUnit.MINUTES)));
+            );
```

Comment:
sure that passes spotless? looks to me like u commited without running the pre-commit hooks.

Reply:
even if it does move it up

Reply:
I forgot spotless I will do it a sec

Reply:
if u have set up ur IDE correctly, u cant forget it. it would auto-execute during commit. its called a pre-commit hook 😄  thanks though